account-kit/react/src/components/auth/card/main.tsx

```diff
@@ -2,15 +2,17 @@ import { Fragment } from "react";
 import { Divider } from "../../divider.js";
 import { PoweredBy } from "../../poweredby.js";
 import { AuthSection } from "../sections/AuthSection.js";
-import type { AuthCardProps } from "./index.js";
 import { ls } from "../../../strings.js";
+import { useUiConfig } from "../../../hooks/useUiConfig.js";
 
 // eslint-disable-next-line jsdoc/require-jsdoc
-export const MainAuthContent = ({
-  header = null,
-  showSignInText = true,
-  sections,
-}: AuthCardProps) => {
+export const MainAuthContent = () => {
+  const { auth } = useUiConfig();
+
+  const header = auth?.header ?? null;
+  const sections = auth?.sections;
+  const showSignInText = auth?.showSignInText ?? false;
```

Comment:
@moldy530 I was thinking that as a follow-up, it might be nice if the `useUiConfig()` hook was where we assigned some defaults, so that some of these properties weren't optional and thus up to the caller to assign a default (e.g. for the `illustrationStyle`)

Reply:
Yeah that's a good idea, like you could pass `ui.auth: true` to create config and get a default setup or something and then the UI config hook just handles the defaults